docs/content/getting-started/quick-start-with-kubernetes.md

```diff
@@ -0,0 +1,321 @@
+---
+title: "Traefik Getting Started Quickly With Kubernetes"
```

Comment:
No need to add "quickly" in the title. A getting started guide implies "quickly"